docs/guides/cli.md

```diff
@@ -1 +1,174 @@
-# Coming Soon
+
+# GuideLLM CLI User Guide 
+
+For more details on setup and installation, see the Setup and [Installation](https://github.com/neuralmagic/guidellm?tab=readme-ov-file#installation) sections. 
+
+## About GuideLLM
+
+The GuideLLM CLI is a performance benchmarking tool to enable you to evaluate and visualize your LLM inference serving performance before your deploy to production. GuideLLM runs via the CLI and takes in a wide array of input arguments to enable you to configure your workload to run benchmarks to the specifications that you desire. This ultimately provides the ability to understand bottlenecks in your inference serving pipeline and make changes before your users are effected by your LLM application. 
```

Comment:
`GuideLLM is a CLI-based performance ..`

before you deploy to production? While in production?

runs via CLI is repetative.

Does GuideLLM take a wide array of user inputs, or does it instead simulate a wide array of user LLM workloads?

Is the workload configured (the thing the user runs) or is the vllm configured here?

"before users are affected" may be better worded as "and improve operational behavior before your users experience your LLM workload.